core/src/test/scala/org/apache/spark/rdd/RDDSuite.scala

```diff
@@ -1129,6 +1129,36 @@ class RDDSuite extends SparkFunSuite with SharedSparkContext {
     }.collect()
   }
 
+  test("SPARK-23496: order of input partitions can result in severe skew in coalesce") {
```

Comment:
this test looks to me as a good candidate for flakiness, since we are are picking random numbers. Can we set the seed in order to avoid this?

Reply:
The test is in fact deterministic. The seed is already fixed here: https://github.com/apache/spark/blob/049f243c59737699fee54fdc9d65cbd7c788032a/core/src/main/scala/org/apache/spark/rdd/CoalescedRDD.scala#L163

Reply:
I see, thanks, sorry, I missed it